server/middleware/validators/validateBookindId.js

```diff
@@ -2,7 +2,7 @@ import resPonse from '../../helpers/responses/response';
 
 const Joi = require('@hapi/joi');
 
-const validateError = (res, result) => resPonse.errorMessage(res, 400, (`${result.error.details[0].context.label}`));
+const validateError = (res, result) => resPonse.errorMessage(res, 400, (`${result.error.details[0].message}`));
```

Comment:
Similar blocks of code in 2 locations here and validateInput.js file, consider refactoring to make the code base DRY.